algebird-core/src/main/scala/com/twitter/algebird/Hash128.scala

```diff
@@ -40,14 +47,17 @@ object Hash128 extends java.io.Serializable {
 
   val DefaultSeed = 12345678L
 
-  def murmur128ArrayByte(seed: Long): Hash128[Array[Byte]] = new Hash128[Array[Byte]] {
```

Comment:
can we keep these `seed` APIs and make the passed `seed` the `defaultSeed` of the instance? I think that will keep the old behavior.